main/src/library/Set.flix

```diff
@@ -281,7 +281,7 @@ namespace Set {
 
     ///
     /// Returns the association set `xs` as a map.
-    /// If `xs` contains multiple values for the same key, the value appearing latest in `xs` is used.
+    /// If `xs` contains multiple values for the same key, the value appearing earliest in `xs` is used.
```

Comment:
My logic here was that in the case of `Set((1, true) :: (1, false) :: Nil)`, the `(1, true)` mapping was inserted into the set last.  Thus, when translated into a `Map`, we could give precedence to the most recently inserted elements in the `Set`, which appear earliest in the underlying `List`.

This is why I did the same with the `List.toMap` method.  Although since we don't have a guaranteed ordering at this point, I'm not sure what would be best, because it is unclear what "earliest" even means.

Should I just change it to "latest" for now (so the latter value takes precedence)?